nasa/harmony-regression-tests
# DAS-1499 - Update HOSS to use EEDTEST RSSMIF16D collection.

### Description:

This PR does three things:

* Replaces the use of RSSMIF16D from the GHRC provider to an EEDTEST mirror. This has become a problem because GHRC changed the permissions on this collection recently.
* Enables these tests for SIT.
* Adds two new HOSS regression tests:
   * Temporal subsetting.
   * Named dimension subsetting.

### Related Tickets:

* DAS-1499 (specifically fix HOSS regression tests)
* HARMONY-1202 (generally fix all UAT/SIT regression tests)

### Test instructions:

* Clone this branch.
* Either:
   * Create a Python environment with all the required Python packages for the notebook.
   * Instantiate that environment.
   *  Start a Jupyter notebook server from within the `test/sds` directory.
   * Run the whole notebook - it should complete without error.
* Or:
   * Use the `Makefile` and `run_notebooks.sh` script to run the notebook in a Docker container (this is most like the Bamboo automated testing).
